server/server.go

```diff
@@ -163,14 +210,14 @@ func NewServer(logger *zap.Logger, storeInstance *store.Store, loggerLevel *zap.
 		s.SchemaSyncer = NewSchemaSyncer(logger, s)
 
 		// Backup runner
-		s.BackupRunner = NewBackupRunner(logger, s, backupRunnerInterval)
+		s.BackupRunner = NewBackupRunner(logger, s, s.profile.BackupRunnerInterval)
```

Comment:
s.profile == prof

If we use prof elsewhere, let's use prof.BackupRunnerInterval consistently here and below.

Reply:
Update.